anymaniax/orval
# fix(msw): make delay optional

## Status

**READY**

## Description

This allows disabling the `delay` call in mocked api calls, which makes the MockHandler sync instead of async.
As per request of Nate on discord

## Todos


## Steps to Test or Reproduce

Make sure delay still can be set by both number and function and that the MockHandler is async.
Try changing the `delay` settings to `false` and make sure the MockHandlers become synchronous and no call to `delay` is made.
